sdk/core/azure-core/CHANGELOG.md

```diff
@@ -4,6 +4,13 @@
 
 ### Features Added
 
+- Added support for HTTP proxy servers, both unauthenticated and with Plain authentication.
+- Added universal support for several TLS options:
+  - Added the ability to set the expected TLS root certificate for TLS connection (useful if a proxy server uses a TLS certificate that is not chained to a known root).
+  - Added the ability to enable TLS certificate revocation list checks (off by default).
+- Added LibCURL functionality for TLS functionality
+  - Allow TLS connection to succeed if CRL retrieval fails.
```

Comment:
Similar feedback from https://github.com/Azure/azure-sdk-for-cpp/pull/3948#discussion_r974694296

This could be a third bullet under "Added universal support for several TLS options"
```suggestion
  - Added TLS functionality for LibCurl (allow TLS connection to succeed if CRL retrieval fails).
```

Reply:
I've improved the quality of the text.